### ISSUE TITLE
custom information keys are nil if derivative cache is enabled

### ISSUE THREAD

**Issue by ghost** [cantaloupe-project/cantaloupe#473]
```
  def my_custom_information_response_keys
    {
      'page_number' => context['page_number'],
      'page_count' => context['page_count']
    }
  end
```

```
cache.server.derivative.enabled = true
cache.server.info.enabled = true
```
or
```
cache.server.derivative.enabled = true
cache.server.info.enabled = false
```
or
```
cache.server.derivative.enabled = false
cache.server.info.enabled = true
```

The first image info request of pdf will include page_count. Subsequent requests that pull from cache do not.

The only option that consistently returns the page_count.

```
cache.server.derivative.enabled = false
cache.server.info.enabled = false
```

**Comment by adolski** [comment]
Thanks for reporting this @wwelling , it is now fixed on the `release/5.0` branch for inclusion in 5.0.1.

**Comment by ghost** [comment]
No problem. Thanks for the quick fix. Was considering looking into myself.
